docs/DeveloperGuide.md

```diff
@@ -1,34 +1,938 @@
+
+Step 7: The result message from `EditModuleCommand#editModule()` is returned to `EditModuleCommand#execute()` and shown to the user by calling `Ui#showToUser()`.
+
+The following sequence diagram shows how the edit module name feature works:
+
+<p align="center">
+  <img src="DG_Images/EditModuleSequenceDiagram.png" width="800" alt="Edit Module Sequence Diagram"/>
+  <br/>Figure <>. Sequence diagram of edit module name feature  
+</p>
+
+>:information_source: <b>Note:</b> The lifeline for `Parser` and `Admin` should end at the destroy marker (X) but due to a limitation of PlantUML, the lifeline reaches the end of diagram.
+
+#### 4.1.4. Remove Module Feature
+(Jia Ern)
+
+The remove module feature allows the user to remove a module by specifying the index of the module in the list. 
+
+The remove module mechanism is facilitated by `RemoveModuleCommand`. It extends from the abstract class `RemoveCommand`. 
+
+In addition, it implements the following operations:
+* `RemoveModuleCommand#execute()` — oversees entire execution for removing a module.
+* `RemoveModuleCommand#removeModule()` — removes module from list of modules including the chapters and flashcards under it.
+* `RemoveCommand#prepareResult()` — prepares the resulting message of the execution.
+
+For instance, the user wants to start a remove the module `CS2113T`, a detailed description of what happens is shown below:
+
+Step 1: The user is currently in `admin` level. 
+
+Step 2: The user enters `remove 1` command to delete the first module in the list of modules — which in this case is `CS2113T`. The `remove` command creates `RemoveModuleCommand` which will then be executed. 
+
+Step 3: `RemoveModuleCommand#execute` gets the `module` based on the index provided and passes it to `Storage#deleteDirectory` to delete the module folder as well as the chapters and flashcards under it. 
+
+The following sequence diagram shows how the remove module feature works:
+![Sequence Diagram of Remove Module](DG_Images/removemod_seq_diagram.png)
+
+#### 4.1.5. Access Module Level Feature
+(Jiayi)
+
+`GoModuleCommand` class facilitates the proposed access module level feature. It extends `GoCommand` class. 
+
+It implements the following operations:
+* `GoModuleCommand#execute()` -- Inherites from the abstract `GoCommand`. It is overrided to calls `GoModuleCommand#goModule()` and print message to user.
+* `GoModuleCommand#goModule()` -- Accesses a new module level from admin level based on the module name entered by user. If the module name does not match with any of the existing module, it returns a error message. Otherwise, this operation returns a successful message to the `GoModuleCommand#execute` operation.
+
+Shown as the class diagram below, with the inheritance of `GoCommand` and `Command`, `Kaji` is able to execute the operation `GoModuleCommand#execute()` directly. 
+
+![Class Diagram of go module command](UML/GoModuleCommandClass.png)
```

Comment:
![image](https://user-images.githubusercontent.com/60380411/97527199-ad433480-19e5-11eb-87e4-058a21366b70.png)
The diagram is not displaying correctly when I look at it. Would be good if you can check and update it!